build/docker/zap-full-scan.py

```diff
@@ -420,6 +424,10 @@ def main(argv):
                 # Save the report
                 write_report(base_dir + report_html, zap.core.htmlreport())
 
+            if report_json:
+                # Save the report
+                write_report(base_dir + report_json, zap.core.jsonreport())
```

Comment:
This was not yet included in the Python client, we need to release a new version with that.